src/main/java/com/gamingmesh/jobs/economy/BufferedEconomy.java

```diff
@@ -220,8 +220,8 @@ public void payAll() {
 		else
 		    Bukkit.getScheduler().runTaskLater(plugin, new BufferedPaymentTask(this, economy, payment), i);
 
-		// Action bar stuff
-		showActionBar(payment);
+		// Show players payment stuff
+			showPayment(payment);

```

Comment:
Try to get rid of whitespaces. I know it's frustrates you, but the github intentiations is not the best since the "commits was created".